new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiUpdateDomainConfig.java

```diff
@@ -685,4 +1006,50 @@ private void createClusterConfigMap(String configMapName, String modelFile) {
     assertTrue(cmCreated, String.format("createConfigMap failed while creating ConfigMap %s", configMapName));
   }
 
+  private ExecResult checkSystemResourceConfiguration(String resourcesType, String resourcesName) {
+
+    int adminServiceNodePort = getServiceNodePort(domainNamespace, adminServerPodName + "-external", "default");
+    ExecResult result = null;
+    curlString = new StringBuffer("status=$(curl --user weblogic:welcome1 ");
+    curlString.append("http://" + K8S_NODEPORT_HOST + ":" + adminServiceNodePort)
+         .append("/management/weblogic/latest/domainConfig")
+         .append("/")
+         .append(resourcesType)
+         .append("/")
+         .append(resourcesName)
+         .append("/")
+         .append(" --silent --show-error ")
+         .append(" -o /dev/null ")
+         .append(" -w %{http_code});")
+         .append("echo ${status}");
+    logger.info("checkSystemResource: curl command {0}", new String(curlString));
+    try {
+      result = exec(new String(curlString), true);
```

Comment:
If you want to refactor the code in the same PR, you can use the below code, change return type to boolean
```
return new Command()
          .withParams(new CommandParams()
              .command(curlString.toString()))
          .executeAndVerify("200");
```
From the calling code, 

```
assertTrue(checkSystemResourceConfiguration("JDBCSystemResources", "TestDataSource"), "JDBCSystemResource is not found");
```

Reply:
In some case I check for 200 and in some cases I check for 404. So I can not generalize the return code to 200 always